README.md

```diff
@@ -28,41 +28,46 @@ Tips on upgrading can be found in [UPGRADE.md](UPGRADE.md)
 
 ## Usage
 
-Using Herodot is straight forward, these examples will help you getting started.
+Using Herodot is straightforward. The examples below will help you get started.
 
 ### JSON
 
-Herodot supplies an interface, so it's possible to write many outputs, such as XML and others. For now, JSON is supported.
+Herodot supplies an interface, allowing to return errors in many data formats like XML and others. Currently, it supports only JSON.
 
 #### Write responses
 
 ```go
-var writer = herodot.NewJSONWriter(nil)
+var hd = herodot.NewJSONWriter(nil)
```

Comment:
`writer.Write` stutters, so I'm looking for a better name for this variable. It doesn't have to be `hd`.

Reply:
I think `hd` makes sense!